### PR TITLE
Observe allocated deltas in system docs

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

A new parameter has been added in the `SHOW STORAGE INFO` that tracks the total unreleased amount of `Delta` objects in the system.

### Pull request type

Please check what kind of PR this is:

- [X] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
[(especially necessary if the PR is related to a release)
](https://github.com/memgraph/memgraph/pull/2066)

Closes:
X

### Checklist:

- [X] Check all content with Grammarly
- [X] Perform a self-review of my code
- [X] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [X] The build passes locally
- [X] My changes generate no new warnings or errors
- [X] Add a corresponding label
- [X] If release-related, add a product and version label
- [X] If release-related, add release note on product PR

### CODE REVIEW

File: pages/database-management/server-stats.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ The result will contain the following fields:
 | vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1. </br> For more info, check out [virtual memory section of the docs](/fundamentals/storage-memory-usage#virtual-memory).                                    |
 | memory_res                   | The non-swapped physical RAM memory a task has used, reported by the OS (in B, KiB, MiB, GiB or TiB).                                                                                                                       |
 | peak_memory_res              | Peak RAM memory usage in the system during the whole run.                                        |
+| unreleased_delta_objects     | Total unreleased number of `Delta` objects in the system.                        |
```
Comment (hal-eisen-MG) [inline pages/database-management/server-stats.md:39]: Avoid using a term to define itself. Rather, explain what it means to be "released" vs "not released" and explain what a "delta object" is

File: pages/database-management/server-stats.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ The result will contain the following fields:
 | vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1. </br> For more info, check out [virtual memory section of the docs](/fundamentals/storage-memory-usage#virtual-memory).                                    |
 | memory_res                   | The non-swapped physical RAM memory a task has used, reported by the OS (in B, KiB, MiB, GiB or TiB).                                                                                                                       |
 | peak_memory_res              | Peak RAM memory usage in the system during the whole run.                                        |
+| unreleased_delta_objects     | The current amount of unreleased objects with the information about the changes that write transactions have made, called Delta objects. Delta objects will eventually be deallocated from the system.                        |
```
Comment (hal-eisen-MG) [inline pages/database-management/server-stats.md:39]: Avoid using a term to define itself. In this case "unreleased".
What are the units for this? KB, GB, MB?

Comment (Josipmrden) [reply pages/database-management/server-stats.md]: just a number of non-deallocated objects

File: pages/database-management/server-stats.md [hunk 3]
```diff
@@ -36,6 +36,7 @@ The result will contain the following fields:
 | vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1. </br> For more info, check out [virtual memory section of the docs](/fundamentals/storage-memory-usage#virtual-memory).                                    |
 | memory_res                   | The non-swapped physical RAM memory a task has used, reported by the OS (in B, KiB, MiB, GiB or TiB).                                                                                                                       |
 | peak_memory_res              | Peak RAM memory usage in the system during the whole run.                                        |
+| unreleased_delta_objects     | The current number of objects non-deallocated objects with the information about the changes that write transactions have made, called Delta objects. Delta objects will eventually be deallocated from the system.                        |
```
Comment (hal-eisen-MG) [inline pages/database-management/server-stats.md:39]: Awkward repetition: "objects non-deallocated objects"

Avoid double negatives such as "non" followed by "de". Perhaps something like "current number of still allocated objects".

Remove "called delta objects".

Please add an answer to the question: "when will they be deallocated?". Saying "eventually" isn't sufficient.

Comment (Josipmrden) [reply pages/database-management/server-stats.md:39]: corrected and added a link where Delta allocation is explained in detail.

**Comment by Josipmrden** [comment]
@kgolubic Please don't merge into memgraph-2-18, we'll wait for 2.19 for this.

**Review by hal-eisen-MG — DISMISSED** [review]
LGTM

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me.

**Pull request merged**